journey-map/public/index.mjs

```diff
@@ -1,7 +1,8 @@
 import mermaid from 'https://cdn.jsdelivr.net/npm/mermaid@10/dist/mermaid.esm.min.mjs';
 import svgPanZoom from 'https://cdn.jsdelivr.net/npm/svg-pan-zoom@3.6.1/+esm';
 import yaml from 'https://cdn.jsdelivr.net/npm/yaml@2.3.2/+esm';
-import { getOptions, render } from './render.mjs';
+import { render } from './render.mjs';
+import {getJourneyContexts, getNestedJourneyStates, getOptions} from "./helpers.mjs";
```

Comment:
should we add prettier (maybe even eslint) to the journey map subfolder? Could do as a separate ticket to keep these changes separate?

Reply:
Yeah, it's definitely big enough to warrant it now...

Definitely a separate PR - if you think it's quick then I'd be happy to review as a 'BAU' change, but otherwise feel free to create a ticket :)